mesonbuild/munstable_coredata.py

```diff
@@ -96,7 +98,8 @@ def run(options):
             native = []
             cross = []
             for dep_key, dep in sorted(v.items()):
-                if dep_key[2]:
+                pprint.pprint(dep_key)
```

Comment:
Stray print?

Reply:
yes. Sorry about that. I force pushed a fixed version.